content/end.js

```diff
@@ -121,6 +123,12 @@ class Logged {
         if (this.tButton && !this.tButton.getAttribute('style') && (this.oldScroll <= window.scrollY || !this.oldScroll))
             this.tButton.style.transform = "scale(1)"
     }
+
+    highlightCode() {
+        document.querySelectorAll('pre, code, tt').forEach((block) => {
+            hljs.highlightBlock(block);
```

Comment:
We sboulh format code only if language relevance is above a threshold.
For example:
```js
let formatted = hljs.highlightAuto(block.innerText)
if (formatted.r > 5) block.innerHTML = formatted.value
block.classList.add('hljs')
```